pkg/kubelet/server/stats/volume_stat_calculator.go

```diff
@@ -123,7 +124,7 @@ func (s *volumeStatCalculator) calcAndStoreStats() {
 			}
 		}
 		volumeStats := s.parsePodVolumeStats(name, pvcRef, metric, volSpec)
-		if isVolumeEphemeral(volSpec) {
+		if util.IsLocalEphemeralVolume(volSpec) {
```

Comment:
At a high level, is there a difference between an "ephemeral" volume and a "local ephemeral" volume? Or is this just a naming distinction?

Reply:
I think it is just a naming distinction, but I'm not confident.

Reply:
For persistent, we could have local or remote. But for ephemeral, we only have local ephemeral.

Reply:
I disagree. A "CSI ephemeral inline" volume could also be remote, if that is how the CSI driver implements the volume. The upcoming "generic ephemeral inline volumes" will use a normal PVC for provisioning, and then the "local" part no longer applies at all.

 I would rename "IsLocalEphemeralVolume" to "IsEphemeralVolume".

Does it matter anywhere whether the volume is local or remote?

Reply:
We clarified this in the definition of IsLocalEphemeralVolume: that the volume is local (= managed by kubelet) does matter.